packages/react-native/Libraries/Text/TextInput/RCTBaseTextInputView.mm

```diff
@@ -377,10 +377,6 @@ - (void)textInputDidBeginEditing
     self.backedTextInputView.attributedText = [NSAttributedString new];
   }
```

Comment:
This makes sense to me. Couldn't immediately get the tester to run - don't know what's changed since last time I ran it - but tested in our app and the results are good.

https://github.com/facebook/react-native/assets/153161762/eb70bd50-cfb5-4b09-9024-f2690ef29540

Interestingly though, I noticed that `onChange` doesn't fire (neither does `onChangeText`) whenever the input clears (present before changes as well). Don't know if that's intended behavior or not, but I feel that I might want to update some state whenever the text gets cleared. Something to note.

Reply:
Thanks for the update @haileyok, I've imported the new changes to see if everything looks good and I'll try to verify the changes on the RN tester, if it's broken we gotta fix it! 🙂